cfy_manager/utils/common.py

```diff
@@ -252,6 +260,13 @@ def is_installed(service):
     return service in config[SERVICES_TO_INSTALL]
 
 
+def get_installed_service_name():
+    """Returns the installed service name. Relevant only for a cluster."""
+    for service_name in DATABASE_SERVICE, QUEUE_SERVICE, MANAGER_SERVICE:
+        if is_installed(service_name):
+            return service_name
```

Comment:
This will not behave as you probably want it to if there are two out of three components installed.

Reply:
Thanks! that also allowed me to think of a better solution!